src/redux/reducers/differentialExpression/differentialExpressionUnselectDeletedOption.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-param-reassign */
+import produce from 'immer';
+import { getCellSetKey } from 'utils/cellSets';
+
+const differentialExpressionUnselectDeletedOption = produce((draft, action) => {
+  const { key: deletedKey } = action.payload;
+
+  Object.keys(draft.comparison.group).forEach((comparisonType) => {
+    const comparisonGroup = draft.comparison.group[comparisonType];
```

Comment:
This could be replaced with 
`Object.entries(draft.comparison.group).forEach(([comparisonType, comparisonGroup]) => {`

Reply:
Ultimately used `Object.values` because we don't need `comparisonType`.